exporters/otlp/otlpmetric/otlpmetrichttp/readme.md

```diff
@@ -0,0 +1,48 @@
+# OpenTelemetry-Go OTLP Metric HTTP Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp)
+
+[OpenTelemetry Protocol Exporter](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/protocol/exporter.md) implementation.
+
+## Installation
+
+```
+go get -u go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp
+```
+
+## Examples
+
+- [HTTP Exporter setup and examples](./example_test.go)
+
+## [`otlpmetrichttp`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp)
+
+Exporters should be created using the [New] and used with a [PeriodicReader].
```

Comment:
Then maybe just `PeriodicReader` and `New` (backticks) instead of hyperlinks 😬

Reply:
Would it ok to leave them as dead links until the release and use https://github.com/open-telemetry/opentelemetry-go/issues/4448 to track updating them after?

Reply:
> Would it ok to leave them as dead links until the release and use #4448 to track updating them after?

I'd rather we don't have dead links so our CI system will pass and therefore be able to catch other link failures.

Reply:
Our CI does pass, just if you click on them they won't take you to the docs page (they redirect to #New or #PeriodicReader on the current page).  If I were to put the URL in behind the PeriodicReader that would cause our CI to fail until a release is made.